openmc/universe.py

```diff
@@ -329,6 +330,8 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
             Axes to draw to
 
             .. versionadded:: 0.13.1
+        legend : bool
+            whether a legend showing material or cell names should be drawn
         **kwargs
             Keyword arguments passed to :func:`matplotlib.pyplot.imshow`
 
```

Comment:
```suggestion
            .. versionadded:: 0.13.1
        **kwargs
            Keyword arguments passed to :func:`matplotlib.pyplot.imshow`

            .. versionadded:: 0.13.4
        legend : bool
            whether a legend showing material or cell names should be drawn
```
Just wondering if this nice new doc string should this be in a new 0.13.4 section

Reply:
I don't see why not, but also do not see why this is particularly necessary given that it's recorded in the git log.